.bazelrc

```diff
@@ -1,6 +1,14 @@
 # import common bazelrc settings used in this repository and in nested workspaces
 import %workspace%/.bazelrc.common
 
+# ts flag aliases
+build --flag_alias=ts_workers=@aspect_rules_ts//ts:supports_workers
```

Comment:
this doesn't seem like a good idea to me unless we are sure that users will define these flag_alias's as well. With this we might read our code and not realize it won't work the same for a user.

Reply:
these are not meant for the users. however, I agree with you that this could be confusing for us especially. perhaps i should remove these?

Reply:
removed them.